Modules/SubModule_Import/xASL_imp_DCM2NII_Subject_ShuffleTheDynamics.m

```diff
@@ -159,7 +159,7 @@
 
     if xASL_exist(x.dir.studyPar,'file')
         studyPar = spm_jsonread(x.dir.studyPar);
-        if isfield(studyPar,'TimeEncodedMatrixSize') && ~isempty(studyPar,'TimeEncodedMatrixSize') || ... % Should be 4, 8 or 12
+        if isfield(studyPar,'TimeEncodedMatrixSize') && ~isempty(studyPar.TimeEncodedMatrixSize) || ... % Should be 4, 8 or 12
```

Comment:
Do we need a warning or feedback here?

Reply:
Let's not change this too much, because it was already modified/renamed etc. in another PR. Keep it like it is for now 👍

Reply:
Okay!